resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/registry/SegmentNode.java

```diff
@@ -74,6 +76,11 @@ public Match(MethodExpression expression, Matcher matcher)
    public ResourceInvoker match(HttpRequest request, int start)
    {
       String path = request.getUri().getMatchingPath();
+
+      LogMessages.LOGGER.pathInfo(request.getUri().getPath());

```

Comment:
is this moved here from SynchronousDispatcher on purpose?

Reply:
Oops. I moved it for testing purpose, and I should put it back to `SynchronousDispatcher`, or the `pathInfo` message will be logged to many times :-) Let me revert this change.